frameworks/GAMA/__init__.py

```diff
@@ -9,11 +9,21 @@ def setup(*args, **kwargs):
 
 def run(dataset: Dataset, config: TaskConfig):
     from frameworks.shared.caller import run_in_venv
+    from frameworks.shared.serialization import ser_config
+    ser_config.pandas_serializer = 'parquet'  # can't use pickle due to pandas version mismatch
```

Comment:
@Innixma just FYI, with this PR, we allow serialization of the pandas dataframes to parquet format to pass data to the subprocess.
I can easily add an option to prevent the deserialization so that `exec.py` can directly consume the parquet files if you're interested.

Reply:
This would be a small improvement before we can directly use the openml parquet file.

Reply:
Or, better: the same way I provide autoconversion to `csv` files which can then be accessed as `dataset.train.data('csv')`, I can do the same with parquet.

Reply:
@Innixma I just tried it, and it seems to work fine for AG.
I will include the change (AG consuming parquet files instead of pandas frame) if you're interested and mention you to look at the changes.

Reply:
This sounds reasonable, the idea would be to pass the parquet string path as `train_data` param to `predictor.fit`. This way the data is loaded internally and there is no duplication of the data in memory. I'm assuming the parquet data loaded via pandas will be identical to the pandas DataFrame currently passed into exec via `dataset.train.data`. If so, this should be ideal.

For test data, it can be loaded from parquet inside of exec.py after `predictor` has been fit, that way test data won't be in memory during `fit`, which should be optimal. While we could pass the parquet string path of test to `predictor.predict`, probably better to just load it prior so we can do things such as drop the label column prior to passing to predict/predict_proba as currently done.